### ISSUE TITLE
getting NCCH error when trying to build the cia

### ISSUE THREAD

**Issue by sapolman** [SirFluffDev/ps1-3ds-forwarder#1]
`ROM    - ./r4.chd`
`ICON   - ./48x48.png`
`BANNER - ./logo.png`
`SOUND  - ./itsshowtime.wav`

`Long name: Ridge Racer Type 4`
`Short name (optional):`
`Publisher: Namco`
`Title ID (optional):` 

`Ridge Racer Type 4 / Ridge Racer Type 4 / Namco / 0xbd9ae`

`Copying ROM`
`Configuring settings`
`Building RomFS`
`Building CIA`
`[NCCH ERROR] Failed to open banner file 'banner.bnr'`
`[NCCH ERROR] NCCH Build Process Failed`
`[RESULT] Failed to build outfile`
`Cleaning up`
`rm: cannot remove 'banner.bnr': No such file or directory`

`Finished (Ridge Racer Type 4.cia)`

on Fedora Workstation 38

**Comment by SirFluffDev** [comment]
Sorry for leaving this unanswered. Were you running from the same directory as the script? If you don't it can cause issues with paths and whatnot.

**Comment by 44ron** [comment]
Exact same issue when trying to convert SOTN, I have the files in the same directory as the script.

**Comment by SirFluffDev** [comment]
But are you in the same directory when you run it? If all your files are in ~/Documents, are you in ~/Documents in your terminal?

**Comment by 44ron** [comment]
yes. i am in the root directory of ps1-3ds-forwarder, where create.sh is. I've placed the files in the root folder and run create.sh there
